tests/generation/test_utils.py

```diff
@@ -2489,10 +2489,10 @@ def test_generate_encoder_outputs_attention_mask(self):
 
     def test_transition_scores_greedy_search(self):
         articles = ["Justin Timberlake", "Michael Phelps"]
-        tokenizer = GPT2Tokenizer.from_pretrained("hf-internal-testing/tiny-random-gpt2")
+        tokenizer = GPT2Tokenizer.from_pretrained("distilgpt2", padding_side="left")
```

Comment:
Changed this one, as the random model had tiny numerical differences that made it select a different token in TF 😠

Reply:
Haha I feel you!